### DISCUSSION TITLE
啊这个怎么登陆啊

### DISCUSSION THREAD

**Discussion by furinayooju** [share121/inter-knot#5107] [绝区零]
求助，怎么登陆和改头像啊

**Comment by furinayooju** [comment]
芙芙保佑，搞定了

**Comment by KELERSAMA** [comment]
登录:"我的"里面把公共接口改成用户接口 
头像:#4990 
这边建议少发帖 多读帖 相似的内容在别的帖子里都能找到 搜索功能也可以用 
<img src="https://bbs-static.miyoushe.com/static/2024/09/10/d60393fe879038382e5c7cdcf7ca25fd_7267933421783671773.png" alt="飞霄 打架" width="75" >

**Comment by furinayooju** [comment]
感谢楼上，主要是在看到app上线一瞬间就去下了，之前没注意这个。抱歉。
